async-http-client/src/test/scala/org/http4s/client/asynchttpclient/AsyncHttpClientSpec.scala

```diff
@@ -10,6 +10,18 @@ package asynchttpclient
 
 import cats.effect.IO
 
-class AsyncHttpClientSpec extends ClientRouteTestBattery("AsyncHttpClient") {
+class AsyncHttpClientSpec extends ClientRouteTestBattery("AsyncHttpClient") with Http4sSpec {
+  implicit val testContext = testContext()
+
   def clientResource = AsyncHttpClient.resource[IO]()
+
+  "AsyncHttpClient configure" should {
```

Comment:
No tests existed at all for `AsyncHttpClient`, I saw what was being done in other test files and tried to replicate it here.

Reply:
I dont think the configuration has a useful equals, so this test is not working

Reply:
I've updated the tests to assert on the fields themselves, rather than the object.